jupyterbook/_config.yml

```diff
@@ -0,0 +1,41 @@
+title: The U.S. Integrated Ocean Observing System (IOOS)
+author: IOOS 
+copyright: "2021"
+logo: images/IOOS-logo_little.png
+
+#only build files that are there in the table of contents. else defaults to building all files 
+only_build_toc_files: true
+
+launch_buttons:
+  notebook_interface: "classic" # The interface interactive links will activate ["classic", "jupyterlab"]
+  binderhub_url: "https://mybinder.org"
+  colab_url: "https://colab.research.google.com"
+  thebe: true
+
+
+#######################################################################################
+# Execution settings
+execute:
+  execute_notebooks           : cache
+
+#######################################################################################
+# HTML-specific settings
+html:
+  home_page_in_navbar         : false
```

Comment:
@lohithmunakala do you mean sending a screen shot with and without this so we can choose? I believe we may want that.

Reply:
#### Below is the web-app with `home_page_in_navbar : true` 
![Screenshot from 2021-06-21 00-51-28](https://user-images.githubusercontent.com/56470584/122686307-eeff4480-d22d-11eb-8e38-bb23c9baaa87.png)

#### Below is the web-app with `home_page_in_navbar : false` 
![Screenshot from 2021-06-21 00-52-24](https://user-images.githubusercontent.com/56470584/122686309-f0307180-d22d-11eb-9991-1eba21113917.png)

Reply:
@ocefpaf, any update regarding this?

Reply:
So, this is simply adding the `IOOS Data Demo Center` link in the navigation on the left? If we keep that in the navbar, we should change the text so it's clear what that is linking to. I'd recommend "Home" or "IOOS Data Demo Center Home".

Reply:
The text will be the name of the homepage. As the name of the homepage is `IOOS Data Demo Center,` it shows up like that. If we rename the homepage to `IOOS Data Demo Center Home` , the text would get renamed automatically. 

We can still go to the homepage by clicking on the IOOS Logo/Name.

Reply:
If clicking on the IOOS logo gets you there, I'd say this is duplicative and can be removed.